src/app/shared/enums/stub-strategy.ts

```diff
@@ -0,0 +1,8 @@
+export let StubStrategyUtil = {
```

Comment:
This is an enum, but the name says this is an util

Reply:
You're right, changed the name